game-service/src/main/java/org/libertybikes/game/core/GameRound.java

```diff
@@ -145,6 +145,8 @@ private void beginLobbyCountdown(Session s, boolean isPhone) {
     }
 
     public void updatePlayerDirection(Session playerSession, InboundMessage msg) {
+        if (msg == null || playerSession == null)
+            return;
```

Comment:
not sure what this is for? If we are sending through null messages or null player sessions I would think that gets caught earlier such as GameRoundWebsocket